docs/source/reverse-engineering/introduction.rst

```diff
@@ -1,6 +0,0 @@
-Introduction
```

Comment:
As far as I can tell this section was axed in the rewrite?

While it doesn't contain any content at all atm, we should have an reverse engineering / native code development section IMO.

Maybe add it as an additional category?

Reply:
I mean, its really just empty so I dont see the point in having it, but if others want it to stay too I will put it back

Reply:
Yeah I know, my point is that we should have the section ready for someone to easily add content to. I'm aware that in it's current state it's useless but not having anything gives the impression that RE stuff is "not supposed to be in modding docs" as opposed to "no one ever wrote stuff for RE".